device/sycl/src/clusterization/clusterization_algorithm.cpp

```diff
@@ -139,11 +143,11 @@ host_spacepoint_container clusterization_algorithm::operator()(
         vecmem::get_data(measurements_prefix_sum), m_queue);
 
     // Copy the results back to the host
-    host_spacepoint_container spacepoints_per_event(&m_mr.get());
-    copy(spacepoints_buffer.headers, spacepoints_per_event.get_headers());
-    copy(spacepoints_buffer.items, spacepoints_per_event.get_items());
+    // host_spacepoint_container spacepoints_per_event(&m_mr.get());
+    // copy(spacepoints_buffer.headers, spacepoints_per_event.get_headers());
+    // copy(spacepoints_buffer.items, spacepoints_per_event.get_items());
```

Comment:
You should just remove these lines completely.

Reply:
thanks, I forgot about this bit